### ISSUE TITLE
Usage directions

### ISSUE THREAD

**Issue by mjrosenb** [mbusb/multibootusb#4]
So, the README says how to install multibootusb, but not how to actually use it.  N.B. I've been using the version in AUR, rather than from github.  It took me a while to figure out that I needed to make my own fat32 partition on the disk, and mount it before it would work.  Then I tried to install an ISO using the first available screen.  This didn't pop up any error message, but after trying to install 3 more .iso's, I noticed I was getting error messages on the console about missing /mnt/multibootusb.   After creating these files, everything installed (and it even popped up a nice message saying it installed successfully!)  after trying to boot into this, and discovering I couldn't, I noticed the "syslinux" tab, which I can only assume is necessary.  Attempting to install everything (second option) spews this on the terminal:
 /dev/sdb1
[root@primer mjrosenb]# multibootusb
Syslinux exist in multibootusb directory.
multibootusb
/dev/sdb1: No such file or directory
Traceback (most recent call last):
  File "/usr/bin/multibootusb", line 699, in onInstall_syslinuxClick
    self.copytree(src, dst)
  File "/usr/bin/multibootusb", line 303, in copytree
    shutil.copy2(s, d)
  File "/usr/lib/python2.7/shutil.py", line 130, in copy2
    copyfile(src, dst)
  File "/usr/lib/python2.7/shutil.py", line 83, in copyfile
    with open(dst, 'wb') as fdst:
IOError: [Errno 2] No such file or directory: '/mnt/multibootusb/extlinux.cfg'
multibootusb

tldr: more error messages are awesome, and how on earth doI actually use this software?

**Comment by mbusb** [comment]
Added user guide in wiki here https://github.com/mbusb/multibootusb/wiki/User-Guide
